__tests__/integration/configuration/docker-compose.yml

```diff
@@ -67,6 +70,166 @@ services:
     networks:
       - hathor-privnet
 
+
+  # Wallet Service section
+  # All the following services are related to the Wallet Service
+  # Redis on 6379, MySQL on 3306
+  # Wallet Service Daemon on 8081 and Daemon Websocket on 8082
+  # Wallet Service API Lambda on 3000 and Wallet Service Websocket Lambda on 3001
+
+  mysql:
+    image: centos/mysql-80-centos7 # Necessary to use specific authentication protocol
+    networks:
+      - hathor-privnet
+    environment:
+      MYSQL_ROOT_PASSWORD: hathor
+      MYSQL_DATABASE: wallet_service
+      MYSQL_USER: wallet_service_user
+      MYSQL_PASSWORD: password
+      MYSQL_DEFAULT_AUTHENTICATION_PLUGIN: mysql_native_password
+    ports:
+      - "3380:3306"
+    healthcheck:
+        test: ["CMD", "mysql", "-h", "mysql", "-u", "wallet_service_user", "-ppassword", "-e", "SELECT 1", "wallet_service"]
+
+  redis:
+    image: redis:6.2
+    networks:
+      - hathor-privnet
+    ports:
+      - "6379:6379"
+    healthcheck:
+      test: [ "CMD", "redis-cli", "ping" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  ws-migrator:
+    image: hathornetwork/hathor-wallet-service-migrator
+    restart: "no"  # Critical: don't restart migration service
+    depends_on:
+      mysql:
+        condition: service_healthy
+    environment:
+      DB_ENDPOINT: "mysql"
+      DB_NAME: "wallet_service"
+      DB_USER: "wallet_service_user"
+      DB_PASS: "password"
+      DB_PORT: 3306
+    networks:
+      - hathor-privnet
+
+  ws-daemon:
+    image: hathornetwork/hathor-wallet-service-sync-daemon
+    depends_on:
+      ws-migrator:
+        condition: service_completed_successfully
+      fullnode:
+        condition: service_healthy
+      mysql:
+        condition: service_healthy
+    environment:
+      # Those two variables are used to set all the necessary dynamic env variables for this pristine environment
+      FETCH_FULLNODE_IDS: true
+
+      # Connection to other services in this private network
+      FULLNODE_HOST: "fullnode:8080"
+      FULLNODE_NETWORK: "nano-testnet-bravo"
+      REDIS_URL: redis://redis:6379
+      DB_ENDPOINT: "mysql"
+      DB_NAME: "wallet_service"
+      DB_USER: "wallet_service_user"
+      DB_PASS: "password"
+      DB_PORT: 3306
+
+      # Other env vars. Some of them may not be necessary and could be removed in the future
+      USE_SSL: "false"
+      NEW_TX_SQS: ""
+      PUSH_NOTIFICATION_ENABLED: "false"
+      WALLET_SERVICE_LAMBDA_ENDPOINT: "" # XXX: Confirm if this is really not used
```

Comment:
I suspect this is used when some parts of the code invokes a Lambda function directly. You can find some cases by looking for `LambdaClient` in the code.

I guess you'll need a fake Lambda server than can mock the behavior of the actual AWS Lambda and use it as the endpoint here.

But Abadesso will know better, I just saw this by chance and wanted to leave my 2 cents.